docker/release/build-image-single-arch.sh

```diff
@@ -123,6 +123,5 @@ else
 fi
 
 # Docker build
-docker build --build-arg VERSION=$VERSION --build-arg BUILD_DATE=`date -u +%Y-%m-%dT%H:%M:%SZ` --build-arg NOTES=$NOTES -f $DOCKERFILE $DIR -t opensearchproject/$PRODUCT:$VERSION
-docker tag opensearchproject/$PRODUCT:$VERSION opensearchproject/$PRODUCT:latest
+docker build --build-arg VERSION=$VERSION --build-arg BUILD_DATE=`date -u +%Y-%m-%dT%H:%M:%SZ` --build-arg NOTES=$NOTES -f $DOCKERFILE $DIR -t opensearchproject/$PRODUCT:$VERSION --build-arg TARGETARCH=${arch_uname}
```

Comment:
Are you using the multi-arch script to test?
The problem with adding a targetarch approach is that you can only do so for a single arch, not multi.
Consider similar to this approach here: https://github.com/opensearch-project/opensearch-build/blob/c0df09424d5a5f997e377a209fd60489c2e48958/docker/release/dockerfiles/opensearch-dashboards.al2.dockerfile#L36

Reply:
And why is the tag step being removed?

Reply:
`uname -m` might be better tho at the time I use `-p` with similar effect.
Tho historically -m should be more portable than -p.
And we have used -m for all other purposes already.

Reply:
I accidentally removed the tag step line. Will add it back.

I have been building using the multi-arch script and its been working. 
```
./build-image-multi-arch.sh -v 1.0.0 -f ./dockerfiles/opensearch.al2.dockerfile -p opensearch -a "x64,arm64" -r "<Docker Hub RepoName>/<Docker Image Name>:<Tag Name>"

and then running using:
$ docker run -it -p 9200:9200 -p 9600:9600 -e "discovery.type=single-node" opensearchproject/opensearch:1.1.0
```

Reply:
I added targetarch there in response to one of your comments on the [previous PR](https://github.com/opensearch-project/opensearch-build/pull/3320#discussion_r1186431181) of this issue.

Reply:
Would your suggestion involve using uname -m inside the docker files instead of targetarch and no changes inside docker/release/build-image-single-arch.sh.

Reply:
I changed to using 'uname -m' now. Tested with single arch and multi arch and both are able to gracefully terminate.